src/plasma/plasma_manager.cc

```diff
@@ -1451,7 +1444,7 @@ void process_object_notification(event_loop *loop,
  * into two structs, one for workers and one for other plasma managers. */
 ClientConnection *ClientConnection_init(PlasmaManagerState *state,
                                         int client_sock,
-                                        char *client_key) {
+                                        const char *client_key) {
```

Comment:
Does it make sense to go further and make `client_key` a `std::string` as well? And then to make `conn->ip_addr_port` a `std::string`? What do you think?

Reply:
Yes, that makes sense. I'll make those updates as well.

Reply:
In fact client_key should be a const std::string& :)

Reply:
Looks like `ClientConnection` is still using the `UT_hash` data structure which can't use `std::string` as a key. I'll need to update that to an ordered map as well. Would you prefer I do that in a separate PR or can I make these changes in here as well?

Reply:
Keeping PRs as small as possible is best, why don't you make a separate one?